under-review/sdfobject-operational_state.sdf.json

```diff
@@ -0,0 +1,56 @@
+{
+  "info": {
+    "copyright": "copyright 2016-2017, 2019 Open Connectivity Foundation, Inc. All rights reserved.",
```

Comment:
Should have copyright 2021 (too)?

Reply:
Not if 2019 was the last time OCF touched it